docs/references/ic-interface-spec.md

```diff
@@ -1944,6 +1945,26 @@ The argument `type` decides which performance counter to return:
 
 In the future, the IC might expose more performance counters.
 
+### Canister limit {#system-api-canister-limit}
+
+The canister can query one of the "canister limits", which denotes a resource limit of the canister imposed by the current configuration of the IC and/or canister. This allows runtime logic, such garbage collector or persistence mechanism, to adapt to the system limits without recompilation and redeployment.
```

Comment:
Do we return the limit (meaning: the constant) or the remaining budget in the current execution?